### ISSUE TITLE
No message received in `test_multiple_figure_closure` with `pyproject.toml` setup

### ISSUE THREAD

**Issue by swvanbuuren** [swvanbuuren/pqthreads#13]
In rare cases, the following error is seen for the test `test_multiple_figure_closure` when build with setup proposed in #12 

Error trace:
```
pqthreads/controllers.py:90: in run
    self.result = self.function(*self.args, **self.kwargs)
tests/basic_test.py:94: in main
    fig1 = worker.figure()
pqthreads/examples/worker.py:42: in figure
    return container.create(**kwargs)
pqthreads/containers.py:125: in create
    self.append(self.item_class(*args, **kwargs))
pqthreads/containers.py:96: in __init__
    self.index = self.agent.create(*args, **kwargs)
pqthreads/agents.py:62: in create
    return self.read_message()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = WorkerAgent(name=figure)

    def read_message(self):
        """ Helper method, that reads message set by slot and returns a copy """
        if self.message is self.no_message:
            if self.error:
                return self.message
                # we don't raise this error anymore, since it's caught in the
                # GUIAgency
                # raise GUIAgentException('Error detected at gui_agent side')
            if not self.error:
>               raise WorkerAgentException('No message received')
E               pqthreads.agents.WorkerAgentException: No message received
```

**Comment by swvanbuuren** [comment]
Fixed with #15

**Comment by swvanbuuren** [comment]
Fixed with #15
